### DISCUSSION TITLE
Remove lines with many styles

### DISCUSSION THREAD

**Discussion by shuniichan** [SubtitleEdit/subtitleedit#7513] [Q&A] [answered]
I want to remove lines with different styles using the batch converter but doesn't seem to find the arguments to input multiple styles at the same time. For example if I want to removes lines with style A and B what should I type in?

**Comment by niksedk** [comment, marked as answer]
In latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.1/SubtitleEditBeta.zip

You can type `A, B`
